app/Http/Controllers/Dormitory/Printing/PrintJobController.php

```
@@ -78,43 +75,48 @@ public function adminIndex()
+        $copyNumber = $validated['copies'];
+        $file = $validated['file'];
+        $originalName = $file->getClientOriginalName();
+        $path = $file->store('', 'printing');
 
-        /** @var Printer */
-        $printer = Printer::firstWhere('name', config('print.printer_name'));
+        $printerConfiguration = PrinterConfiguration::find($validated['printer_configuration']);
 
-        $path = $file->store('', 'printing');
-        $originalName = $file->getClientOriginalName();
         $pageNumber = PrinterHelper::getDocumentPageNumber(Storage::disk('printing')->path($path));
```

PHPMD (2.15.0)

Avoid unused local variables such as '$pageNumber'. (Unused Code Rules)

(UnusedLocalVariable)